test/sql/cluster.sql

```diff
@@ -34,9 +34,35 @@ CLUSTER VERBOSE;
 -- Change the clustered index
```

Comment:
Bringing up another comment about tests:
1) We need toast fields to test this. See `toast_test` table in size_utils test.
2) We need a test for the dropped toast-column test. (Forcing pSwapByContent = false)
3) We need to test both seq and index scans after each cluster. An example
```
BEGIN;
  SET LOCAL enable_seqscan='off';
  EXPLAIN (costs off) SELECT * FROM cluster_test;
  SELECT * FROM cluster_test;
COMMIT;
```
The first line turn off seq scans, forcing an indexscan (to turn of indexscans the option is enable_indexscan). The second line prints the plan to check it's what we want. The third line executes the plan.
4) (optional) Right now we have some tests that do `using index scan` and some that do `using sequential scan and sort` But this seems to happen by accident. I think a test that explicitly does each would be better practice. I am not positive but I think adjusting `random_page_cost` option could force the plan from one to the other.